src/Stack/Types/Config.hs

```diff
@@ -891,8 +891,16 @@ parseConfigMonoidObject rootDir obj = do
     configMonoidSaveHackageCreds <- First <$> obj ..:? configMonoidSaveHackageCredsName
     configMonoidHackageBaseUrl <- First <$> obj ..:? configMonoidHackageBaseUrlName
     configMonoidIgnoreRevisionMismatch <- First <$> obj ..:? configMonoidIgnoreRevisionMismatchName
-    configMonoidColorWhen <- First <$> obj ..:? configMonoidColorWhenName
-    configMonoidStyles <- fromMaybe mempty <$> obj ..:? configMonoidStylesName
+
+    configMonoidColorWhenPrimary <- obj ..:? configMonoidColorWhenName
+    configMonoidColorWhenAlternative <- obj ..:? configMonoidColorWhenAlternativeName
+    let configMonoidColorWhen =  First $   configMonoidColorWhenPrimary
+                                       <|> configMonoidColorWhenAlternative
+
+    configMonoidStylesPrimary <- obj ..:? configMonoidStylesName
+    configMonoidStylesAlternative <- obj ..:? configMonoidStylesAlternativeName
+    let configMonoidStyles = fromMaybe mempty $   configMonoidStylesPrimary
+                                              <|> configMonoidStylesAlternative
```

Comment:
This (from line 894) would be clearer if rather than `Primary` and `Alternative` the options were `US` and `GB` (or equivalent); it wasn't obvious that `Primary` and `Alternative` referred to _spellings_.

Reply:
Done - I now use `US` and `GB` consistently.